include/swift/AST/ClangModuleLoader.h

```diff
@@ -124,7 +124,7 @@ class ClangModuleLoader : public ModuleLoader {
   using ModuleLoader::ModuleLoader;
 
 public:
-  virtual clang::TargetInfo &getTargetInfo() const = 0;
+  virtual clang::TargetInfo &getModuleAvailabilityTarget() const = 0;
```

Comment:
Do you think that it would make sense to a comment here and with the `getTarget()` API explaining the difference (similar to the commit message)

Reply:
Yes, absolutely, will add a comment.